internal/eval/commands.go

```diff
@@ -686,3 +686,26 @@ func init() {
 	DiceCmds["PFADD"] = pfAddCmdMeta
 	DiceCmds["PFCOUNT"] = pfCountCmdMeta
 }
+
+func convertKeySpecsToSlice(keySpecs KeySpecs) []interface{} {
```

Comment:
Shall we keep parity with redis output? For `COMMAND` e.g. our output for `HELLO`
```
0.0.0.0:7379> command
 1) 1) "HELLO"
    2) (integer) -1
    3) 1) "BeginIndex"
       2) (integer) 0
       3) "LastKey"
       4) (integer) 0
       5) "Step"
       6) (integer) 0
```
Redis output:
```
205) 1) "hello"
     2) (integer) -1
     3) 1) "noscript"
        2) "loading"
        3) "stale"
        4) "fast"
        5) "no_auth"
        6) "allow_busy"
     4) (integer) 0
     5) (integer) 0
     6) (integer) 0
     7) 1) "@fast"
        2) "@connection"
     8) (empty array)
     9) (empty array)
     10) (empty array)
```
Redis output follows below format:

1. Name
2. Arity
3. Flags
4. First key
5. Last key
6. Step
7. [ACL categories](https://redis.io/operate/oss_and_stack/management/security/acl) (as of Redis 6.0)
8. [Tips](https://redis.io/develop/reference/command-tips.md) (as of Redis 7.0)
9. [Key specifications](https://redis.io/develop/reference/key-specs.md) (as of Redis 7.0)
10. Subcommands (as of Redis 7.0)

We don't have all the fields set but for the ones we do have available we should maintain parity.
More details you can found [here](https://redis.io/docs/latest/commands/command/).

Reply:
Yes @lucifercr07 have tried to maintain parity with redis output for COMMAND command, so with this PR the output is currently

1. Name
2. Arity
3. Key Specifications

But I'm guessing, Begin Index, LastKey, Step in Key Specifications maps to number 4.(First Key), 5.(Last key), 6. (Step) respectively in the Redis output format you pasted above.  

Will that be a correct understanding?

Reply:
Yes that's correct. We just need to remove `1) "BeginIndex"`, `3) "LastKey"`, `5) "Step"` from our output.

Reply:
Got it, awesome!

Reply:
Updated the same @lucifercr07 

The output for the COMMAND command now looks like

`16) 1) "HELLO"
    2) (integer) -1
    3) (integer) 0
    4) (integer) 0
    5) (integer) 0`